### PR TITLE
Using shell commands in a IPython cell

### PR DESCRIPTION
**Pull request by hofmannedv**

### CODE REVIEW

File: docs/workspace/ipython/shell.ipynb [hunk 1]
```diff
@@ -11,7 +11,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The IPython Notebook allows a whole range of UNIX/Linux commands. These are presented below as an overview. To begin with, the command `ls` lists the files in the current working directory:"
+    "The IPython Notebook allows simple UNIX/Linux commands to be
+executed in a single input cell. There are no limits but when using,
+please keep in mind that in contrast to a regular UNIX/Linux shell,
+start each shell command with a `!`, for example `!ls` for the command
+`ls` (see below for further explanations about the command).
+Furthermore, each shell command is executed in its own subshell. For
```
Comment (veit) [inline docs/workspace/ipython/shell.ipynb:19]: This is described in more detail in workspace/ipython/unix-shell/file-system.ipynb. Possibly the notebook shell.ipynb should be deleted and the contents integrated into workspace/ipython/unix-shell/.

File: docs/workspace/ipython/shell.ipynb [hunk 2]
```diff
@@ -11,7 +11,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The IPython Notebook allows a whole range of UNIX/Linux commands. These are presented below as an overview. To begin with, the command `ls` lists the files in the current working directory:"
+    "The IPython Notebook allows simple UNIX/Linux commands to be
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/workspace/ipython/shell.ipynb]: I fixed that

File: docs/workspace/ipython/shell.ipynb [hunk 3]
```diff
@@ -81,6 +112,15 @@
     "contents = !ls"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here the variable `contents` has been assigned the output of the
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/workspace/ipython/shell.ipynb]: I added this

File: docs/workspace/ipython/shell.ipynb [hunk 4]
```diff
@@ -31,6 +40,13 @@
     "!ls"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The command `!pwd` displays the current working directory:"
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/workspace/ipython/shell.ipynb:40]: fixed

File: docs/workspace/ipython/shell.ipynb [hunk 5]
```diff
@@ -81,6 +112,15 @@
     "contents = !ls"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here the variable `contents` has been assigned the output of the
+command `ls`. As a result, `contents` is a list, where each list element
+corresponds to a line in the output. With the `print` command you output the list contents:"
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/workspace/ipython/shell.ipynb]: A good idea -- ia changed this

File: docs/workspace/ipython/shell.ipynb [hunk 6]
```diff
@@ -11,7 +11,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The IPython Notebook allows a whole range of UNIX/Linux commands. These are presented below as an overview. To begin with, the command `ls` lists the files in the current working directory:"
+    "The IPython Notebook allows simple UNIX/Linux commands to be
+executed in a single input cell. There are no limits but when using,
```
[the remark this answers is not in the document]
Comment (hofmannedv) [reply docs/workspace/ipython/shell.ipynb]: you are right -- let us think about how to express that properly

**Pull request merged**